src/tasks/tasks.py

```diff
@@ -1558,43 +1558,63 @@ def get_metrics(self, reset=False):
         acc = self.scorer1.get_metric(reset)
         return {'accuracy': acc}
 
-    def process_split(self, split, indexers) -> Iterable[Type[Instance]]:
-        ''' Process a tagging task '''
-        inputs = [TextField(list(map(Token, sent)),
-                            token_indexers=indexers) for sent in split[0]]
-        targs = [TextField(list(map(Token, sent)), token_indexers=self.target_indexer)
-                 for sent in split[2]]
-        # Might be better as LabelField? I don't know what these things mean
-        instances = [Instance({"inputs": x, "targs": t})
-                     for (x, t) in zip(inputs, targs)]
-        return instances
-
     def get_all_labels(self) -> List[str]:
         return self.all_labels
 
-
 @register_task('ccg', rel_path='CCG/')
 class CCGTaggingTask(TaggingTask):
     ''' CCG supertagging as a task.
         Using the supertags from CCGbank. '''
-
-    def __init__(self, path, max_seq_len, name, **kw):
+    def __init__(self, path, max_seq_len, name="ccg", **kw):
         ''' There are 1363 supertags in CCGBank. '''
-        super().__init__(name, num_tags=1363, **kw)
+        super().__init__(name, 1363, **kw)
+        self.INTRODUCED_TOKEN = '1362'
+        self.bert_tokenization = 1 if kw['tokenizer_name'].startswith("bert") else 0
```

Comment:
Use `self._tokenizer_name.startswith("bert-")` instead of looking in `kw`?

Also don't need the 1 if ... else 0, should be fine to just store the bool.